docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -292,6 +292,17 @@ Sets the maximum time using a duration format (5s/5m/5ms) before timing out read
 
 <hr />
 
+## [server.custom_response_headers]
+
+This setting enables you to specify additional headers that the server adds to HTTP(S) responses.
+
+```
+; exampleHeader1 = exampleValue1
+; exampleHeader2 = exampleValue2
+```
```

Comment:
I'm not sure a commented-out example is that useful in the doc. Maybe remove it, or replace with an uncommented example?

```suggestion
```